src/WebJobs.Script/Utility.cs

```diff
@@ -144,6 +145,14 @@ public static string GetFunctionShortName(string functionName)
             return functionName;
         }
 
+        // "Namespace.Class.Method" --> "Namespace.Class"
+        public static string GetFullClassName(string fullFunctionName)
```

Comment:
FunctionEntryPointResolver  doesn't do the relevant operation. 

---
In reply to: [129905588](https://github.com/Azure/azure-webjobs-sdk-script/pull/1717#discussion_r129905588) [](ancestors = 129905588)